src/applications/coronavirus-vaccination/components/Introduction.jsx

```diff
@@ -98,6 +120,9 @@ function Introduction({ isLoggedIn, toggleLoginModal }) {
 const mapStateToProps = state => {
   return {
     isLoggedIn: userSelectors.isLoggedIn(state),
+    authButtonEnabled: !toggleValues(state)[
+      FEATURE_FLAG_NAMES.covidVaccineUpdatesDisableAuth
+    ],
```

Comment:
Small nit - do we need to worry about flicker at all, like when the flags are being loaded up? While the browser is loading the feature flag state from flipper, `authButtonEnabled` will be `!null` which will become `true` but then Flipper might finish loading and set it to `!true`. By the time this component renders the feature flags may already be available anyway, so many not matter.